### DISCUSSION TITLE
亲 瑞幸咖啡的领取9.9的广告麻烦老板去除下 获取不到ID

### DISCUSSION THREAD

**Discussion by wndlsj** [gkd-kit/gkd#202] [General]
快照，只能获取整个页面的ID

https://gkd-kit.gitee.io/snapshot/1698204611265

**Comment by lisonge** [comment]
快照链接是错误的

**Reply by wndlsj** [reply]
https://zhazhayun.lanzout.com/b0etvjknc
密码:27jf

麻烦您看一下 这个快照为什么是全选页面 不能单独选择小框呢
{"id":1698211595463,"appId":"com.lucky.luckyclient","activityId":"com.luckin.client.main.FirstActivity","appName":"瑞幸咖啡","appVersionCode":155,"appVersionName":"5.1.21","screenHeight":1920,"screenWidth":1080,"isLandscape":false,"device":{"device":"sagit","model":"MI 6","manufacturer":"Xiaomi","brand":"Xiaomi","sdkInt":27,"release":"8.1.0"},"nodes":[{"id":0,"pid":-1,"index":0,"attr":{"id":null,"name":"android.widget.FrameLayout","text":null,"desc":null,"clickable":false,"focusable":false,"checkable":false,"checked":false,"visibleToUser":true,"left":0,"top":0,"right":1080,"bottom":1920,"width":1080,"height":1920,"childCount":2,"index":0,"depth":0}},{"id":1,"pid":0,"index":0,"attr":{"id":null,"name":"android.widget.LinearLayout","text":null,"desc":null,"clickable":false,"focusable":false,"checkable":false,"checked":false,"visibleToUser":true,"left":0,"top":0,"right":1080,"bottom":1920,"width":1080,"height":1920,"childCount":1,"index":0,"depth":1}},{"id":2,"pid":0,"index":1,"attr":{"id":"android:id/statusBarBackground","name":"android.view.View","text":null,"desc":null,"clickable":false,"focusable":false,"checkable":false,"checked":false,"visibleToUser":false,"left":0,"top":0,"right":1080,"bottom":72,"width":1080,"height":72,"childCount":0,"index":1,"depth":1}},{"id":3,"pid":1,"index":0,"attr":{"id":"android:id/content","name":"android.widget.FrameLayout","text":null,"desc":null,"clickable":false,"focusable":false,"checkable":false,"checked":false,"visibleToUser":true,"left":0,"top":0,"right":1080,"bottom":1920,"width":1080,"height":1920,"childCount":1,"index":0,"depth":2}},{"id":4,"pid":3,"index":0,"attr":{"id":null,"name":"android.widget.LinearLayout","text":null,"desc":null,"clickable":false,"focusable":false,"checkable":false,"checked":false,"visibleToUser":true,"left":0,"top":0,"right":1080,"bottom":1920,"width":1080,"height":1920,"childCount":1,"index":0,"depth":3}},{"id":5,"pid":4,"index":0,"attr":{"id":"com.lucky.luckyclient:id/webview_dialog","name":"android.widget.FrameLayout","text":null,"desc":null,"clickable":false,"focusable":false,"checkable":false,"checked":false,"visibleToUser":true,"left":0,"top":0,"right":1080,"bottom":1920,"width":1080,"height":1920,"childCount":1,"index":0,"depth":4}},{"id":6,"pid":5,"index":0,"attr":{"id":null,"name":"android.widget.RelativeLayout","text":null,"desc":null,"clickable":false,"focusable":false,"checkable":false,"checked":false,"visibleToUser":true,"left":0,"top":0,"right":1080,"bottom":1920,"width":1080,"height":1920,"childCount":1,"index":0,"depth":5}},{"id":7,"pid":6,"index":0,"attr":{"id":"com.lucky.luckyclient:id/webcontent","name":"android.webkit.WebView","text":null,"desc":null,"clickable":true,"focusable":true,"checkable":false,"checked":false,"visibleToUser":true,"left":0,"top":0,"right":1080,"bottom":1920,"width":1080,"height":1920,"childCount":0,"index":0,"depth":6}}]}
<img src="https://wx1.sinaimg.cn/mw2000/69a84ad1ly1hj7ghzcb4oj21yu0yvtgc.jpg" >

**Reply by lisonge** [reply]
![image](https://github.com/gkd-kit/gkd/assets/38517192/baa79b3c-d169-4de2-9a10-11e0cef2152b)

**Reply by wndlsj** [reply]
https://zhazhayun.lanzout.com/b0etvjknc
密码:27jf
因为不能传文件就放在网盘啦

**Reply by lisonge** [reply]
https://github.com/gkd-kit/subscription/issues/1406
